Header header
int32 height
int32 width
int32 max_y
int32 min_y
uint8[] intensity
uint16[] distance
